modules/wms/package.json

```diff
@@ -41,8 +41,9 @@
     "@babel/runtime": "^7.3.1",
     "@loaders.gl/loader-utils": "3.3.0-alpha.10",
     "@loaders.gl/schema": "3.3.0-alpha.10",
-		"@turf/rewind": "^5.1.5",
-		"deep-strict-equal": "^0.2.0"
+    "@loaders.gl/xml": "3.3.0-alpha.10",
```

Comment:
@jwasilgeo This should fix the issue you reported

Reply:
great thanks